bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -565,7 +559,7 @@ protected void submitCallback(int code) {
                 }
             }
             LOG.error("Read of ledger entry failed: L{} E{}-E{}, Heard from {}. First unread entry is {}",
-                    new Object[] { lh.getId(), startEntryId, endEntryId, heardFromHosts, firstUnread });
+                    new Object[] { lh.getId(), startEntryId, endEntryId, heardFromHostsBitSet, firstUnread });
```

Comment:
@jvrao I added the heardFromHosts set back in e93a5cb . so the error messages will include the host information. let me know if it is okay to you.